package.json

```diff
@@ -17,7 +18,9 @@
     "package": "cross-env NODE_ENV=production node -r babel-register -r babel-polyfill webpack/package.js",
     "package-all": "npm run package -- --all",
     "postinstall": "node node_modules/fbjs-scripts/node/check-dev-engines.js package.json",
+    "preinstall": "which electron || npm install electron@1.4.12",
```

Comment:
Hmm … seems like the only option then. I just hope we won't run into "the latest electron version of nix is behind the real one but has a bug and we cannot upgrade" problems 😉 e.g: we are using `1.4.15` at the moment.